feeds/npm/npm.go

```diff
@@ -75,19 +50,49 @@ func fetchPackages() ([]*Package, error) {
 	return rssResponse.Packages, nil
 }
 
-// Gets the package version from the NPM.
-func fetchVersionInformation(packageName string) (string, error) {
-	resp, err := httpClient.Get(fmt.Sprintf("%s/%s", versionURL, packageName))
+// Gets the package version & corresponding created date from NPM.
+func fetchVersionInformation(pkg *Package) (*Package, error) {
+	resp, err := httpClient.Get(fmt.Sprintf("%s/%s", versionURL, pkg.Title))
 	if err != nil {
-		return "", err
+		return nil, err
 	}
 	defer resp.Body.Close()
-	v := &PackageVersion{}
-	err = json.NewDecoder(resp.Body).Decode(v)
+	body, err := ioutil.ReadAll(resp.Body)
 	if err != nil {
-		return "", err
+		return nil, err
+	}
+	var jsonMap map[string]interface{}
+	err = json.Unmarshal(body, &jsonMap)
+	if err != nil {
+		return nil, err
+	}
+	distTags, ok := jsonMap["dist-tags"].(map[string]interface{})
+	if !ok {
+		return nil, errJSON
```

Comment:
We can wrap this with some additional detail referencing the field we failed to extract.